lib/Doctrine/ORM/Tools/Console/Command/SchemaTool/UpdateCommand.php

```diff
@@ -39,6 +40,11 @@
 class UpdateCommand extends AbstractCommand
 {
     /**
+     * @var SymfonyStyle
+     */
+    protected $io;
```

Comment:
Maybe `ui` or `userInterface` is a better name?

Reply:
Also, wouldn't it be better to have this property in the `AbstractCommand`

Reply:
I can change this ofc, but I use `$io` everywhere and it was mentioned in the official blog post, too. `$ui` is a bit missleading IMO

@javiereguiluz what do you think, how do you use it?

Official blog post: https://symfony.com/blog/new-in-symfony-2-8-console-style-guide

Reply:
I just suggested that because IMO we're actually interacting with the CLI user interface 😄